src/components/panel/panel.js

```diff
@@ -489,6 +482,16 @@ angular
 
 /**
  * @ngdoc method
+ * @name MdPanelPosition#tryPanelPositions
```

Comment:
Can we make the array for the users so that they can follow the same format as `withPanelPosition`?

```
new MdPanelPosition()
    .relativeTo('myButton')
    .tryPanelPosition(xPosition.ALIGN_END, yPosition.ALIGN_TOPS)
    .tryPanelPosition(xPosition.OFFSET_START, yPosition.ALIGN_TOPS)
    .tryPanelPosition(xPosition.CENTER, yPosition.CENTER);
```

I think that `withPanelPosition` is really an alias for `tryPanelPosition` but with only one option.

I'm also afraid the user will end up writing the same xPosition and try the vertical ones, and then only trying the horizontal ones if something fails, which makes a lot of duplication here. Basically, if a user "just wants the panel centered and on screen", they end up trying a lot of options before they find something that works. Essentially, I don't know if there's enough feedback to say "The panel fits in the xPosition, so I will only try to change the vertical ones."